js/bitflyer.js

```diff
@@ -205,7 +205,7 @@ module.exports = class bitflyer extends Exchange {
         const symbol = this.safeSymbol (undefined, market);
```

Comment:
@kroitor  since for this case the market has to be in the request I'm not "fetching" it again because it was in the fetchTicker to construct the request

Reply:
Yes, but we still have to pass it through parseTicker (2nd arg) and to safeTicker (2nd arg).

Reply:
From fetchTicker to parseTicker was already being passed, only missing in the safeTicker